app/models/concerns/account_avatar.rb

```diff
@@ -18,7 +18,7 @@ def avatar_styles(file)
 
   included do
     # Avatar upload
-    has_attached_file :avatar, styles: ->(f) { avatar_styles(f) }, convert_options: { all: '+profile "!icc,*" +set modify-date +set create-date' }, processors: [:lazy_thumbnail]
+    has_attached_file :avatar, styles: ->(f) { avatar_styles(f) }, convert_options: { all: '+profile "!icc,*" +set date:modify +set date:create +set date:timestamp' }, processors: [:lazy_thumbnail]
```

Comment:
I would keep `+set modify-date` and `+set create-date` unless we can identify which ImageMagick versions output these and make sure they are not in use anymore:
```suggestion
    has_attached_file :avatar, styles: ->(f) { avatar_styles(f) }, convert_options: { all: '+profile "!icc,*" +set modify-date +set create-date +set date:modify +set date:create +set date:timestamp' }, processors: [:lazy_thumbnail]
```

Reply:
Did some further testing, it seems like the old flags are broken on ImageMagick versions 7.1.1 (latest release), 6.9.10 (used on Ubuntu 20.04), and 6.9.11 (used on current Debian/Ubuntu), but do indeed work with 6.9.12 (latest legacy). The new flags seem to work on all versions listed above, so I don't see the old flags being of much use here.